homeassistant/components/pi_hole/manifest.json

```diff
@@ -3,5 +3,6 @@
   "name": "Pi-hole",
   "documentation": "https://www.home-assistant.io/integrations/pi_hole",
   "requirements": ["hole==0.5.1"],
-  "codeowners": ["@fabaff", "@johnluetke"]
+  "codeowners": ["@fabaff", "@johnluetke"],
```

Comment:
@shenxn 

Thank you for your contribution thus far! 🎖 Since this is a significant contribution, we would appreciate you'd added yourself to the list of code owners for this integration. ❤️ 

Please, add your GitHub username to the `manifest.json` of this integration.

For more information about "code owners", see: [Architecture Decision Record 0008: Code owners](https://github.com/home-assistant/architecture/blob/master/adr/0008-code-owners.md).